node/src/GlideClient.ts

```diff
@@ -154,7 +191,52 @@ export class GlideClient extends BaseClient {
         this.configurePubsub(options, configuration);
         return configuration;
     }
-
+    /**
+     * Creates a new `GlideClient` instance and establishes a connection to a standalone Valkey Glide server.
+     *
+     * @param options - The configuration options for the client, including server addresses, authentication credentials, TLS settings, database selection, reconnection strategy, and Pub/Sub subscriptions.
+     * @returns A promise that resolves to a connected `GlideClient` instance.
+     *
+     * @remarks
+     * Use this static method to create and connect a `GlideClient` to a standalone Valkey Glide server. The client will automatically handle connection establishment, including any authentication and TLS configurations.
+     *
+     * ### Example - Connecting to a Standalone Server
+     * ```typescript
```

Comment:
```suggestion
     * @example
     * ```typescript
     * // Connecting to a Standalone Server
```